src/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -27,10 +27,9 @@ public static IEnumerable<object[]> ProcessorArchitectures_TestData()
         public static IEnumerable<object[]> Names_TestData()
         {
             yield return new object[] { "name", "name" };
-            yield return new object[] { "NAME", "NAME" };
```

Comment:
Rather than not having any names that involve capitals, should we instead do an ignore-case comparison in the test? (I'm actually asking; not sure which is better)